objects/applications/data_hub/dh-prod-jupyterhub.yaml

```diff
@@ -0,0 +1,17 @@
+#---
```

Comment:
Why is this commented out?

Reply:
Commented out temporarily so you can see how it looks for prod, but will test it out for stage/monitoring first.